web/src/components/MemoContent/index.tsx

```diff
@@ -26,13 +26,15 @@ interface Props {
   onDoubleClick?: (e: React.MouseEvent) => void;
 }
 
+type compactModes = "more" | "less" | "none";
```

Comment:
* types should always be upper cases.

```suggestion
type ContentCompactView = "ALL" | "SNIPPET";
```

Reply:
Changes applied